swayfmt/tests/mod.rs

```diff
@@ -1608,6 +1608,40 @@ impl MyContract for Contract {
     );
 }
 
+#[test]
+fn configurable_comments() {
+    check(
+        r#"
+script;
+
+use std::{constants::ZERO_B256, vm::evm::evm_address::EvmAddress};
+
+configurable { /* inline comment */
+    // double slash comment
+    /// triple slash comment
+    SIGNER: EvmAddress = EvmAddress {
+        value: ZERO_B256,
```

Comment:
it would be also useful to test adding comments inside of structs and enums in configurable blocks, and at the end of the line.

```suggestion
        // comment here
        // multiline!
        value: ZERO_B256, // end of line too
```